src/platform-implementation-js/dom-driver/gmail/gmail-element-getter.ts

```diff
@@ -225,7 +225,7 @@ const GmailElementGetter = {
         ?.getAttribute('aria-expanded') === 'true' ??
       // Default to true if we can't find the element because
       // the majority of users seem to favor that version.
-      false
+      true
```

Comment:
Defaulting to true I'm afraid is the action that will break things. I am happy to talk about it more offline.